packages/langchain/src/openai/embedding.ts

```diff
@@ -13,7 +13,7 @@ export class AzureOpenAiEmbeddingClient
   extends Embeddings
   implements AzureOpenAiEmbeddingModelParams
```

Comment:
The purpose is that when we re-generate or change the type, the class also breaks, otherwise we might notice it too late.
This is less of an issue for the embedding class as the types are much more straightforward, but this can happen quickly in the chat class.